cheatsheets/PHP_Configuration_Cheat_Sheet.md

```diff
@@ -111,3 +107,9 @@ report_memleaks         = On
 track_errors            = Off
 html_errors             = Off
 ```
+
+## Suhosin
+
+[Suhosin](http://www.hardened-php.net/suhosin/index.html) is a patch to PHP which provides a number of hardening and security features that are not available in the default PHP build. However, Suhosin only works with PHP 5, which is **unsupported** and **should not be used**.
+
+There are a number of projects that have attempted to implement similar features for PHP 7 such as [Snuffleupagus](https://snuffleupagus.readthedocs.io/) and [Suhosin-ng](https://github.com/sektioneins/suhosin-ng/wiki/News), but these are both in the pre-release stage, and as such **should not be used in production**.
```

Comment:
*attempting* would be more appropriate as both projects are work in progress at this point. attempted feels like attempted but never succeeded and then project closed.

Reply:
@rbsec Let me know what you think. For me, this is not a blocker, can be amended definitely.

Reply:
Hi both,

I think `have attempted` is more accurate, as there have also been other projects such as [suhosin7](https://github.com/sektioneins/suhosin7) which didn't make huge progress and have subsequently been abandoned.

Reply:
fair enough.